lib/takeaway.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'order'
+
+class Takeaway
+
+  attr_accessor :active_order, :order_items, :menu_item, :kitchen, :cooking
+
+    def initialize
+      @kitchen = Kitchen.new
+        menu
+    end
+
+    def menu
+
+      #displays the menu
+
+      @menu_item = [
+        {id: 1, name: 'Dish 1', price: 6.50},
+        {id: 2, name: 'Dish 2', price: 8.50},
+        {id: 3, name: 'Dish 3', price: 8.00} ]
+
+      puts "============================== MENU =============================="
+
+      puts "#{@menu_item[0][:id]}. #{@menu_item[0][:name]}.........#{@menu_item[0][:price]}"
+      puts "#{@menu_item[1][:id]}. #{@menu_item[1][:name]}.........#{@menu_item[1][:price]}"
+      puts "#{@menu_item[2][:id]}. #{@menu_item[2][:name]}.........#{@menu_item[2][:price]}"
+
+      @menu_item.each do | key, value |
+        puts key
+        end
+    end
+
+    def create_order(name)
+      @active_order = Order.new(name)
+    end
+
+    def test_method(dish_id)
+
+      chosen_dish = @menu_item.select {|dish| dish[:id] == dish_id}
+
+      dish = chosen_dish.pop
+
```

Comment:
is this line necessary? could you just put @active_order.order_items << chosen_dish.pop on line 44 instead?